crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -1,6 +1,7 @@
 pub mod blockscout;
 pub mod ethplorer;
 pub mod liquidity;
+pub mod seasolver;
```

Comment:
nit: Again, I wouldn't call this `seasolver`. While they are the ones with exotic tokens and want to provide a "token owners API", we should name it more generically, as potentially other solvers may want to use the same interface for providing token owner addresses for exotic tokens.